lib/web_ui/lib/src/ui/semantics.dart

```diff
@@ -275,6 +275,123 @@ class SemanticsFlag {
   }
 }
 
+class AttributedString {
```

Comment:
I wonder if there is a way to avoid this duplication?
This class has to be in dart:ui so that i can write the class specific dartconverter. If we put it in framework side, we will need to go back to the List<Object?> thing with the std::any thing we did before.

Reply:
Yes, we end up duplicating classes from dart:ui into the web implementation.

The good part about doing it here is we can think a bit more about how it impacts web, if at all. It may make sense to throw an unimplemented error until this is implemented, but it'd be worth checking with @ferhatb or @yjbanov about that.

Reply:
If it's simply copy&paste, then it's fine. That's how things work as of right now. There are ideas to automate it using some script, but we can directly share code because `dart2js` doesn't understand the `native` keyword and native wrapper classes. Since the Flutter team is the only non-Dart SDK user of the `dart:*` namespace it is unlikely to be fixed.

New API should not throw exceptions. Otherwise, someone could start using this API in one of the standard widgets, such as `ElevatedButton` and break all of Flutter Web. Instead, we should implement it, or provide a graceful fallback/polyfill.